reana/reana_benchmark/cli.py

```diff
@@ -95,6 +95,42 @@ def _to_range(workflow_range: str) -> (int, int):
     type=str,
     callback=lambda c, p, v: _to_range(v),
 )
+
+
+def _to_datetime_range(
+    datetime_range: Optional[str],
+) -> Union[Tuple[str, str], None, NoReturn]:
+    """Take datetime range string, split it into left and right datetime limits, validate and return."""
+    if not datetime_range:
+        return None
+
+    # middle position between two datetimes, position value is always the same
+    separator_position = 19
```

Comment:
Minor: perhaps we could declare a constant for this in the config?
Same for `length_of_two_datetimes_plus_separator`. I think we could name it `DATETIME_LENGTH` and `DATETIME_RANGE_LENGTH`? WDYT?

Reply:
Not sure it needs to go to `config`. It is only used in this function. Probably I should uppercase constants in `_to_datetime_range` to make specific that they are constants.

Reply:
It looks like you are not allowed to define constant (uppercase) variables inside the function so I moved it to `config.py`